server/database/index.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  try {
+    console.log(typeof process.env.MONGO_URI, process.env.MONGO_URI)
```

Comment:
remove this console log. You probably don't want to be logging this :) 

Also, we should consider if you want to use a formal logging library. In python, I would recommend this, would be good for someone to research the topic

Reply:
Yes, I meant to delete this. Updating now